generators/app/templates/src/containers/Profile/components/Image/image.component.js

```diff
@@ -55,7 +55,7 @@ export const Image = ({ webId, defaultProfilePhoto }: Props) => {
     try {
       const { user } = data;
       await user.vcard_hasPhoto.set(namedNode(uri));
-      successToaster(t('profile.uploadSuccess'));
+      successToaster(t('profile.uploadSuccess'), t('profile.successTitle'));
     } catch (error) {
       errorToaster(error.message, 'Error');
```

Comment:
Title should be translated instead of hardcoded

Reply:
Done